Makefile

```diff
@@ -1,5 +1,5 @@
 PROJECT_NAME := Pulumi SDK
-PROJECT_ROOT := $(realpath .)
+PROJECT_ROOT := ..
```

Comment:
I'm surprised this works, since `.` is very different then `..`.

Reply:
This is indeed not pretty. It's just we run `GO_TEST` from subfolder like `pkg`, So relative path has to be `pkg/..`.

With realpath the idea was to use an absolute path instead. But something became flaky on how Windows computes that.

Reply:
I think we should add a comment or maybe just replace `PROJECT_ROOT` with `..`. As it is, this is actively confusing since `PROJECT_ROOT` is neither stable nor the project root.